web/playground/src/examples.js

```diff
@@ -1,28 +1,31 @@
 const examples = {
   "introduction.prql": [
     "sql",
-    `from invoices
-filter invoice_date >= @1970-01-16
-derive {                        # This adds columns
-  transaction_fees = 0.8,
-  income = total - transaction_fees  # Columns can use other columns
-}
-filter income > 1     # Transforms can be repeated.
-group customer_id (   # Use a nested pipeline on each group
-  aggregate {         # Aggregate each group to a single row
-    average total,
-    sum_income = sum income,
-    ct = count sum_income,
+    `from invoices                        # PRQL query begins with a table
+derive {                             # "derive" adds columns
+  transaction_fees = 0.8,            # "=" sets a column name
```

Comment:
```suggestion
  transaction_fee = 0.8,             # "=" sets a column name
```
singular column names?